pkg/agent/proxy/proxier.go

```diff
@@ -605,7 +758,7 @@ func (p *proxier) GetServiceFlowKeys(serviceName, namespace string) ([]string, [
 		svcFlows := p.ofClient.GetServiceFlowKeys(svcInfo.ClusterIP(), uint16(svcInfo.Port()), svcInfo.OFProtocol, epList)
 		flows = append(flows, svcFlows...)
 
-		groupID, _ := p.groupCounter.Get(svcPortName)
+		groupID, _ := p.groupCounter.Get(svcPortName, false)
```

Comment:
I don't think this is correct. This causes antctl command not output incomplete groups.
I'm fine with fixing it later, but there should be at least a TODO or an issue to track it.